hubstorage/client.py

```diff
@@ -49,7 +49,8 @@ class HubstorageClient(object):
     RETRY_DEFAULT_JITTER_MS = 500
     RETRY_DEFAULT_EXPONENTIAL_BACKOFF_MS = 500
 
-    def __init__(self, auth=None, endpoint=None, connection_timeout=None, max_retries=None, max_retry_time=None):
+    def __init__(self, auth=None, endpoint=None, connection_timeout=None,
+                 max_retries=None, max_retry_time=None, useragent=None):
```

Comment:
Just a note (implement it if you want :smile:  ): perhaps it'd be better if `useragent` was named `user_agent`, to follow the notation used in the other arguments.